openslides/core/migrations/0006_auto_20160922_1126.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.10.1 on 2016-09-22 09:26
+from __future__ import unicode_literals
+
+import uuid
+
+from django.db import migrations, models
+from openslides.core.models import Projector
+
+
+def add_speaker_overlay(apps, schema_editor):
+    """
+    Adds a list of speaker overlay to the existing projector id=1
+    """
+    # We disconnect the inform_changed_data_receiver to disabled the
+    # autoupdate mechanism during migrations.
+    models.signals.post_save.disconnect(
+        dispatch_uid='inform_changed_data_receiver')
+
+    # We get the model from the versioned app registry;
+    # if we directly import it, it will be the wrong version.
+    projector = Projector.objects.get(id=1)
+    soverlay_uuid = uuid.uuid4().hex
+    projector.config[soverlay_uuid] = {
+        'name': 'core/speakeroverlay',
+        'stable': True}
+    projector.save()
+
```

Comment:
You have to reconnect autoupdate. See users/migrations/0004_groups.py for example.